src/features/camera.md

```diff
@@ -0,0 +1,240 @@
+
+It is possible to implement your own [custom camera
+projections][cb::camera-custom-projection]. This can give you full control over
+the coordinate system. However, beware that things might behave in unexpected
+ways if you violate Bevy's [coordinate system conventions][cb::coords]!
+
+## HDR and Tonemapping
+
+[See here!][cb::hdr]
+
+## Render Target
+
+The render target of a camera determines where the GPU will draw things to. It
+could be a window (for outputting directly to the screen) or an
+[`Image`][bevy::Image] [asset][cb::asset] (render-to-texture).
+
+By default, cameras output to the primary window.
+
+```rust,no_run,noplayground
+{{#include ../code/src/features/camera/general.rs:render-target}}
+```
+
+## Viewport
+
+The viewport is an (optional) way to restrict a camera to a sub-area of its
+render target, defined as a rectangle. That rectangle is effectively treated as
+the "window" to draw in.
+
+An obvious use-case are split-screen games, where you want a camera to only draw
+to one half of the screen.
+
+```rust,no_run,noplayground
+{{#include ../code/src/features/camera/general.rs:set-viewport}}
+```
+
+If you need to find out the area a camera renders to (the viewport, if
+configured, or the entire window, if not):
+
+```rust,no_run,noplayground
+{{#include ../code/src/features/camera/general.rs:get-viewport}}
+```
+
+## Coordinate Conversion
+
+[`Camera`][bevy::Camera] provides methods to help with coordinate conversion
+between on-screen coordinates and world-space coordinates. For an example, see
+the ["cursor to world"][cookbook::cursor2world] cookbook page.
+
+## Clear Color
+
+This is the "background color" that the whole viewport will be cleared to,
+before a camera renders anything.
+
+You can also disable clearing on a camera, if you want to preserve all the
+pixels as they were before.
+
+[See this page for more info.][cb::clearcolor]
+
+## Camera Ordering
+
+A camera's priority is a simple integer value that controls the order relative
+to any other cameras with the same render target.
+
+For example, if you have multiple cameras that all render to the primary window,
+they will behave as multiple "layers". Cameras with higher priority will render
+"on top of" cameras with lower priority.
+
+```rust,no_run,noplayground
+{{#include ../code/src/features/camera/general.rs:overlay}}
+```
+
+## UI Rendering
+
+Bevy UI rendering is integrated into the cameras! Every camera will, by default,
+also draw UI.
+
+Old versions of Bevy required a separate camera to be spawned for the UI. For the
+sake of a simpler user experience, the UI is now automatically drawn.
+
+However, if you are working with multiple cameras, you probably only want your
+UI to be drawn once (probably by the main camera). You can UI rendering on your
+other cameras.
+
+Also, UI on multiple cameras is currently broken in Bevy. Even if you want
+multiple UI cameras (say, to display UI in an app with multiple windows), it
```

Comment:
Link to issue could be good.

Reply:
I couldn't find any single issue that covers this nicely. The information seems to be scattered across multiple issues and PRs.